src/inference/include/ie/streams_executor.hpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
include/ie is a folder with public API 1.0
We need to move inference/src

Reply:
updated